README.md

```diff
@@ -280,3 +283,31 @@ We use [Mono's Coding Guidelines](http://www.mono-project.com/community/contribu
 
 We use [Bugzilla](https://bugzilla.xamarin.com/enter_bug.cgi?product=Android) to track issues.
 
+# Maintainer FAQ
+
+## How do I rebuild Mono Native Binaries?
+
+The various Mono runtimes -- all *22* of them (!) -- all store object code
+within `build-tools/mono-runtimes/obj/$(Configuration)/TARGET`.
+
+If you change sources within `external/mono`, a top-level `make`/`xbuild`
+invocation may not rebuild those mono native binaries. To explicitly rebuild
+Mono for a given target, run `mono` from the relevant directory.
+For example, to rebuild Mono for armeabi-v7a:
+
+	$ cd build-tools/mono-runtimes
+	$ make -C obj/Debug/armeabi-v7a
+	
+	# This updates bin/$(Configuration)/lib/xbuild/Xamarin/Android/lib/armeabi-v7a/libmonosgen-2.0.so
+	$ xbuild /t:_InstallRuntimes
+
+## How do I rebuild BCL assemblies?
+
+The Xamarin.Android Base Class Library assemblies, such as `mscorlib.dll`,
+are built within `external/mono`, using Mono's normal build system:
+
+	# This updates external/mono/mcs/class/lib/monodroid/ASSEMBLY.dll
+	$ make -C external/mono/mcs/class/ASSEMBLY PROFILE=monodroid
```

Comment:
Instead of this I find it easier to just mention `make -C build-tools/mono-runtimes/obj/Debug/host-Darwin` since that'll ensure all class libs are built/updated correctly